.github/workflows/cross-merge.yaml

```diff
@@ -0,0 +1,12 @@
+name: Ibexa Reusable / Cross Org PR
```

Comment:
@ibexa-yuna Is it ok? Do we need this in this package?

Reply:
It can be anywhere. Also there's no other way.